interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -172,14 +172,13 @@
 <html>
   <head>
     <title><?php echo xlt('Send a Reminder') ?></title>
+    <?php call_required_libraries(array("jquery-min-3-3-1","font-awesome", "iziModalToast"));?>
```

Comment:
This looks nicer and cleaner @Gichecha. Hope you enjoyed the git tutorial i gave you. Welcome on board. I am approving this. Please delete the other PR

Reply:
Oh please edit your PR title. I know you are super excited it worked. Lets keep things professional :)

Reply:
Done...thanks